data/records/delphi-conditions-database.json

```diff
@@ -0,0 +1,75 @@
+[
+  {
+    "abstract": {
+      "description": "The DELPHI conditions database."
+    },
+    "accelerator": "CERN-LEP",
+    "collaboration": {
+      "name": "DELPHI collaboration",
+      "recid": "80050"
+    },
+    "collections": [
+      "DELPHI"
+    ],
+    "date_created": [
+      "2000"
+    ],
+    "date_published": "2025",
+    "distribution": {
+      "formats": [
+        "zebra"
```

Comment:
The `distribution.formats` field is technical and usually indicates the file extension. I would use `dat` here, and explain about Zebra in the abstract?